frontend/server.js

```diff
@@ -118,6 +118,7 @@ async function main() {
               const response = await getOrCreateUser({
                 authId: profile.sub,
                 name: profile.displayName,
+                emailAddress: profile.emails,
```

Comment:
I think `emails` is an array, so this should be:

```suggestion
                emailAddress: profile.emails[0],
```

Reply:
Ah. That might explain why the login flow on https://fnhs-dev-david.westeurope.cloudapp.azure.com/auth/login fails to complete after I deployed this branch.